narratives/ncov_sit-rep_2020-03-27.md

```diff
@@ -0,0 +1,2734 @@
+* [Updates for Oceania](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=17).  
+* [What you can do](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=19).  
+* [Scientific credit](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=20).  
+
+<!-- There is no right side text -->
+
+
+<!-- ##### slide break ##### -->
+<!-- This is left-side text 12-->
+# [Primarily travel-related cases sampled in South America](https://nextstrain.org/ncov/2020-03-27?c=division_exposure&d=tree,map&f_region=Central%20America,South%20America&m=div&p=full)
+We don't have much data from South America yet; of those data, most are returned travelers.
+<br><br>
+While there are some potentially interesting patterns starting to emerge in the non-travel-associated cases, we don't yet have enough evidence to draw conclusions.
+Sequencing efforts are ramping up, and we will report back as the situation becomes clearer.
+<!-- There is no right side text -->
+
+
+
+
+<!-- ############ ASIA ############# -->
+<!-- This is left-side text 13-->
+# [Updates for Asia](https://nextstrain.org/ncov/2020-03-27?d=tree,map&f_region=Asia&p=grid&r=country&c=country&legend=closed)
+
+* [Executive summary](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-26?n=1).
+* [Background resources](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-26?n=2).     
+* [About this data](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=3).  
+* [Updates for Europe](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=4).  
+* [Updates for North America](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=7).  
+* [Updates for Central & South America](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=11).  
+* [Updates for Asia](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=13).  
+* [Updates for Africa](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=15).  
+* [Updates for Oceania](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=17).  
+* [What you can do](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=19).  
+* [Scientific credit](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-27?n=20).  
+
+<!-- There is no right side text -->
+
+
+<!-- ##### slide break ##### -->
+<!-- This is left-side text 14-->
+# [Possible second transmission chain within Iran](https://nextstrain.org/ncov?c=division_exposure&label=clade:A3&m=div&d=tree,map&f_division_exposure=Iran)
+<!-- There is no right side text -->
+
+We do not yet have viral genomes from Iran.
+However, we do have sequences from many cases with reported travel to Iran.
+<br><br>
+Most of the Iranian-linked cases nest within the clade shown here.
+<br><br>
+At the very bottom, we see a new sequence from a woman in Pakistan with reported travel to Iran.
+Interestingly, this case is an outgroup to all of the other cases that we know about from Iran.
```

Comment:
```suggestion
```

Reply:
The sequence from Pakistan seems to be derived from earlier cases of the outbreak in Iran